### PR TITLE
Including text moderation - toxicity classification lab, and the other minor fixes.

### PR DESCRIPTION
**Pull request by lanazhang**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: content-moderation/04-text-moderation/02-content-moderation-text-toxic-classification.ipynb [hunk 1]
```diff
@@ -0,0 +1,473 @@
...
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# First, let's get the latest installations of our dependencies\n",
+    "%pip install -qU pip\n",
+    "%pip install botocore -qU\n",
```
Comment (matteofigus) [inline content-moderation/04-text-moderation/02-content-moderation-text-toxic-classification.ipynb]:
```suggestion
```

File: content-moderation/04-text-moderation/02-content-moderation-text-toxic-classification.ipynb [hunk 2]
```diff
@@ -0,0 +1,473 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import boto3\n",
+    "import botocore\n",
```
Comment (matteofigus) [inline content-moderation/04-text-moderation/02-content-moderation-text-toxic-classification.ipynb]:
```suggestion
```

File: content-moderation/04-text-moderation/02-content-moderation-text-toxic-classification.ipynb [hunk 3]
```diff
@@ -0,0 +1,473 @@
...
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# install pandas and numpy if you are using a local IDE and they are not installed in your env\n",
+    "%pip install pandas\n",
+    "%pip install numpy"
```
Comment (matteofigus) [inline content-moderation/04-text-moderation/02-content-moderation-text-toxic-classification.ipynb]:
```suggestion
    "# install pandas if you are using a local IDE and they are not installed in your env\n",
    "%pip install pandas"
```
I see no use of numpy

File: content-moderation/04-text-moderation/02-content-moderation-text-toxic-classification.ipynb [hunk 4]
```diff
@@ -0,0 +1,473 @@
...
+   "outputs": [],
+   "source": [
+    "import boto3\n",
+    "import botocore\n",
+    "import sagemaker as sm\n",
+    "import os\n",
+    "import io\n",
+    "import datetime\n",
+    "import pandas as pd\n",
+    "import numpy as np\n",
```
Comment (matteofigus) [inline content-moderation/04-text-moderation/02-content-moderation-text-toxic-classification.ipynb]:
```suggestion
```

File: content-moderation/04-text-moderation/02-content-moderation-text-toxic-classification.ipynb [hunk 5]
```diff
@@ -0,0 +1,473 @@
...
+    "s3=boto3.client('s3')\n",
+    "comprehend=boto3.client('comprehend', region_name=region)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Step 2: Prepare custom classification training dataset <a id=\"step2\"></a>\n",
+    "Unzip the sample data **toxicity.zip** and decompress files to a local folder"
```
Comment (matteofigus) [inline content-moderation/04-text-moderation/02-content-moderation-text-toxic-classification.ipynb:102]:
```suggestion
    "Unzip the sample data **toxicity.zip** and decompress files to a local folder. The dataset contained in **../datasets/toxicity.zip** is an unaltered redistribution of [the toxicity dataset](https://github.com/surge-ai/toxicity) made available by Surge AI under MIT License."
```

**Review by matteofigus — CHANGES_REQUESTED** [review]
a couple of nit-picks

**Review by matteofigus — APPROVED** [review]

**Review by ConnorKirk — APPROVED** [review]

**Pull request merged**